app/lib/features/chat/widgets/custom_input.dart

```diff
@@ -2,17 +2,15 @@ import 'dart:io';
 
 import 'package:acter/common/models/types.dart';
```

Comment:
Overall changes in this class looks to me still if @gtalha07 can review this once then it will be good as he might have better understanding of this class.